internal/message.go

```diff
@@ -27,4 +27,7 @@ const (
 
 	// UnhandledMessage represents the default message for unhandled errors
 	UnhandledMessage = "Something went wrong. Please try again. If the problem persists, contact our customer support."
+
+	// Error message when the server is a virtual location, but user has virtual-location off
+	SpecifiedServerIsVirtualLocation = "please enable virtual location access to connect to this server"
```

Comment:
Lowercase and without punctuation at the end? :thinking:  The rest of the messages are uppercase and with punctuation